api-tester/src/main/java/com/revenuecat/apitester/kotlin/StoreTransactionAPI.kt

```diff
@@ -24,6 +24,7 @@ private class StoreTransactionAPI {
             val su1: String? = storeUserID
             val purchaseType: PurchaseType = purchaseType
             val subscriptionOptionId = subscriptionOptionId
+            val prorationMode = prorationMode
```

Comment:
Oh I just noticed we are not specifying the type here, so this wouldn't actually catch issues with the type. This and the above should have the type, like this:
`val prorationMode: ProrationMode? = prorationMode`